wolfcrypt/src/asn.c

```diff
@@ -33322,7 +33338,12 @@ int wc_SetDatesBuffer(Cert* cert, const byte* der, int derSz)
 int EncodePolicyOID(byte *out, word32 *outSz, const char *in, void* heap)
 {
     word32 idx = 0, nb_val;
-    char *token, *str, *ptr;
+#ifdef WOLFSSL_NO_MALLOC
+    char str[1024];
```

Comment:
Hardcoded 1024B : same as the key usage. Let's have a define for this

Reply:
`asn.c` changes wound up being unneeded -- reverted.